tron/config/config_parse.py

```diff
@@ -270,6 +272,61 @@ class ValidateSecretSource(Validator):
 valid_secret_source = ValidateSecretSource()
 
 
+def valid_permission_mode(value, config_context):
```

Comment:
super-nit:
```suggestion
def valid_permission_mode(value: Union[str, int], config_context: ConfigContext) -> str:
```

that said: thoughts on also add some validation either in paasta or in the soaconfigs hooks/tests?

Reply:
I was thinking about adding it in yelpsoa pre-commit, as that will be the best place for developers to catch secret_volumes misconfiguration